addons/binding/org.openhab.binding.rfxcom/ESH-INF/thing/channels.xml

```diff
@@ -5,6 +5,18 @@
 
 	<!-- Channel definitions -->
 
+    <channel-type id="rawmessage">
```

Comment:
I had spaces instead of tabs. I guess the xml doesn't autoformat, fixed.

This is something that probably needs adding to the automatic eclipse config. The editor is set up with spaces for tabs, so I can't add a tab manually, but the formatter changes them on request.